### DISCUSSION TITLE
알라딘 노드 output 과대

### DISCUSSION THREAD

**Discussion by tyl1996** [Meoseon-Division/node-red-contrib-ssafy-meoseon#11] [Ideas]
https://github.com/Meoseon-Division/node-red-contrib-ssafy-meoseon/blob/e2b3b4f0be7a0afae084aef346f54482d1fc6736/Meoseon/Aladin.js#L19

params에 msg가 모두 들어가서 너무 커지는 것 같습니다

input
- parts
- payload
- _msgid

output
- _msgid
- parts
- payload
- params
  - 원래 원하는 데이터
  - parts
  - payload
  - _msgid

**Comment by keelim** [comment]
```js
node.options = {
    headers: {
        'X-Naver-Client-Id': node.clientId,
        'X-Naver-Client-Secret': node.clientSecret,
    },
    params: {
        'query': item,
        'display': config.display,
        'start': config.start,
        'sort': config.sort,
    },				
};
```

- NaverBooks 노드에서 위와 같은 방식으로 assign 하고 있는데 참고하시면 좋을 것 같습니다.
